Source/SwiftLintFramework/Rules/Idiomatic/ReduceIntoRule.swift

```diff
@@ -0,0 +1,35 @@
+import SourceKittenFramework
+
+public struct ReduceIntoRule: ConfigurationProviderRule, OptInRule, AutomaticTestableRule {
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static var description = RuleDescription(
+        identifier: "reduce_into",
+        name: "Reduce Into",
+        description: "Prefer `reduce(into:_:)` over `reduce(_:_:)`",
+        kind: .performance,
+        minSwiftVersion: .three,
+        nonTriggeringExamples: [
+            "let matches = lines.reduce(into: []) { matches, line in",
```

Comment:
can you make sure that the examples are valid Swift code (e.g. the closures are also valid)?